pandas/io/tests/test_pytables.py

```diff
@@ -4733,6 +4733,109 @@ def test_invalid_complib(self):
         with ensure_clean_path(self.path) as path:
             self.assertRaises(ValueError, df.to_hdf, path, 'df', complib='blosc:zlib')
 
+    # GH10447
+    def test_complex(self):
```

Comment:
let's start a new class here (I have started splitting up HDFStore testing like that for other cases).